<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelfCheckResultShowDialogClass</class>
 <widget class="QDialog" name="SelfCheckResultShowDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>自检结果显示</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../favicon.ico</normaloff>../../../favicon.ico</iconset>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>完成 </string>
   </property>
   <property name="btnStyle" stdset="0">
    <string>normalBtnRed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>自检结果显示</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1051</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>——————————————————————————————————————————————————————————————————————————————————————————</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>971</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="checkBox_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="checkBox_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_7">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="checkBox_8">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="checkBox_9">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox_10">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="checkBox_11">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QCheckBox" name="checkBox_12">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
